src/content/translations/hr/eth2/shard-chains/index.md

```diff
@@ -48,15 +48,15 @@ Kada se isporuče prvi lanci djelića, oni će samo pružiti dodatne podatke mre
 
 Rollup je tehnologija „sloja 2” koja postoji danas. Omogućuju aplikacijama dapp da grupiraju ili skupe transakcije u jednu transakciju izvan lanca, generiraju kriptografski dokaz i zatim ga predaju lancu. To smanjuje podatke potrebne za transakciju. Kombinirajući to s dodatnom dostupnošću podataka koju pružaju djelić dobivate 100 000 transakcija u sekundi.
 
-[Više o tehnologiji rollup](/developers/docs/layer-2-scaling/)
+[Više o tehnologiji rollup](/developers/docs/scaling/layer-2-rollups/#rollups)
 
 ## Lanci djelića, verzija 2: izvršavanje koda {#code-execution}
 
 Plan je uvijek bio dodati dodatnu funkcionalnost djelićima, kako bi postali sličniji današnjoj [ glavnoj mreži Ethereum ](/glossary/#mainnet). To bi omogućilo pohranu i izvršavanje pametnih ugovora i upravljanje računima. No, s obzirom na pojačanje u broju transakcija u sekundi koje pružaju djelići verzije 1, je li to potrebno? O tome se još uvijek raspravlja u zajednici i čini se da postoji nekoliko mogućnosti.
 
 ### Trebaju li djelići izvršenje koda? {#do-shards-need-code-execution}
 
-Vitalik Buterin, u podcastu Bankless, predstavio je 3 potencijalne opcije o kojima vrijedi razgovarati. <iframe width="100%" height="315" src="/developers/docs/layer-2-scaling/" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen mark="crwd-mark"></iframe>
+Vitalik Buterin, u podcastu Bankless, predstavio je 3 potencijalne opcije o kojima vrijedi razgovarati. <iframe width="100%" height="315" src="/developers/docs/scaling/#layer-2-scaling" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen mark="crwd-mark"></iframe>
```

Comment:
Looks like this was a bug previously, but the `src` here should be the YouTube embed link: https://www.youtube.com/embed/-R0j5AMUSzA?start=5841
```
Vitalik Buterin, u podcastu Bankless, predstavio je 3 potencijalne opcije o kojima vrijedi razgovarati. <iframe width="100%" height="315" src="https://www.youtube.com/embed/-R0j5AMUSzA?start=5841" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen mark="crwd-mark"></iframe>
```